nrt/data/IEEE14/IEEE14_WithAutomata/description/IEEE14_WithAutomata.tex

```diff
@@ -106,7 +109,249 @@ \section{Results}
 \caption{Behavior of the under voltage automaton}
 \end{figure}
 
-This corresponds to the following timeline:
-\lstinputlisting[breaklines, frame = lines, framesep = 2em]{../IEEE14_UnderVoltageAutomaton/reference/outputs/timeLine/timeline.log}
+
+\newpage
+\section{Phase Shifter I}
+\label{PhaseShifterIAutomaton}
+
+The phase-shifter I is an automaton that gradually changes the tap of a phase-shifter transformer if its monitored transiting current goes higher than a defined threshold for a certain amount of time.
+
+\subsection{Initial Conditions}
+
+Initial conditions are the same that in the IEEE14 basic test cases.
+
+\subsection{Models}
+
+Instead of a ratio tap changer transformer (no phase shift) between buses 5 and 6, we use a phase tap changer transformer controlled by a phase-shifter I automaton in order to regulate the transiting current.
+The automaton is supposed to keep the transit within acceptable bounds, by reducing or increasing the phase shift step by step. \\
+
+The phase-shifter I measures the current flowing through a transformer and sends an order to change its tap if the monitored transiting current stays higher than $I_{Max}$ for a certain amount of time. Then the automaton keeps on making the transformer change its tap until its current reach $I_{Stop}$. The first tap is changed after a delay $t_{1st}$ and the next taps are changed after $t_{Next}$ seconds. \\
+
+The phase-shifter parameters are:
+\begin{center}
+\begin{tabular}{l|l|l}
+   $I_{Max}=0.475p.u$ & $t_{1st}=10s$ & $tap_{Max}=13$ \\
+   $I_{Stop}=0.45p.u$  & $t_{Next}=5s$ & $tap_{Min}=1$ \\
+\end{tabular}
+\end{center}
+
+\subsection{Scenario}
+At $t=20s$, the following line disconnection is simulated:
+\begin{itemize}
+\item{the line connecting bus 9 to bus 10 is opened at its bus 10 end}
+\end{itemize}
+
+\subsection{Solver}
+The solver used is the variable time step solver IDA with the following parameters:
+\begin{center}
+\begin{tabular}{l|l|l}
+   $Order$=2 & $Accuracy_{Rel}$=10e-4 & $Accuracy_{Abs}$=10e-4 \\
+\end{tabular}
+\end{center}
+
+\newpage
+\subsection{Results}
+
+At t=20s, the line is disconnected. Following the topology modification, the current through the transformer monitored by the phase shifter I becomes higher than its threshold $I_{Max}$. After a time delay of 10s, the phase shifter changes its first step, which results in a step decrease of the transformer's current. Then, step are changed every 5s and the process repeats until the current goes back inside the acceptable range, i.e. until it reaches $I_{Stop}$. \\
+
+\begin{figure}[H]
+\subfigure[Transformer's current (p.u.)]
```

Comment:
In the graph it seems that the phase shifter stops acting before the current reaches iStop...